src/main/kotlin/no/nav/familie/ef/sak/infrastruktur/config/FeatureToggleConfig.kt

```diff
@@ -47,6 +47,10 @@ class FeatureToggleConfig(private val enabled: Boolean,
             override fun isEnabled(toggleId: String, defaultValue: Boolean): Boolean {
                 return unleash.isEnabled(toggleId, defaultValue)
             }
+
+            override fun shutdown() {
```

Comment:
Men hvem trigger denne? Er det noe spring-magi jeg ikke kan? 🤔

Reply:
Kanskje legge til en kommentar om at det er spring som kaller denne automagisk ved shutdown? :)

Reply:
Enig med sonar om den tomme destroy metoden: "Add a nested comment explaining why this function is empty"